src/components/QualityEvaluation/QualityEvaluationForm.tsx

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Form, Formik, useField } from "formik";
+import { CSSProperties, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { Item, Indicator } from "@radix-ui/react-radio-group";
+import { ButtonV2 } from "@ndla/button";
+import { colors, spacing, fonts, misc } from "@ndla/core";
+import { FieldErrorMessage, Fieldset, InputV3, Label, Legend, RadioButtonGroup } from "@ndla/forms";
+import { RevisionMetaFormType } from "../../containers/FormikForm/AddRevisionDateField";
+import { formatDateForBackend } from "../../util/formatDate";
+import { FormControl, FormField } from "../FormField";
+import validateFormik, { RulesType } from "../formikValidationSchema";
+
+export const qualityEvaluationOptions: { value: number; color: string }[] = [
+  { value: 1, color: colors.support.green },
+  { value: 2, color: "#C3D060" },
+  { value: 3, color: colors.support.yellow },
+  { value: 4, color: "#CF9065" },
+  { value: 5, color: colors.assessmentResource.dark },
+];
+
+const StyledFieldset = styled(Fieldset)`
+  display: flex;
+  gap: ${spacing.xsmall};
+  align-items: center;
+`;
+
+// Color needed in order for wcag contrast reqirements to be met
+const blackContrastColor = "#000";
+
+export const gradeItemStyles = css`
+  padding: 0px ${spacing.nsmall};
+  background-color: var(--item-color);
+  font-weight: ${fonts.weight.semibold};
+  border-radius: ${misc.borderRadius};
+  color: ${blackContrastColor};
+  ${fonts.size.text.content};
```

Comment:
Kan du sette textStyle direkte på elementer istedenfor å bruke disse?

Reply:
Jeg kan det, grunnen til at jeg gjorde det sånn var for å slippe et ekstra Text-wrapper element inne i radio button, i tillegg til å kun trenge å ha den delte stylingen et sted!